core/tests/protractor_utils/TopicEditorPage.js

```diff
@@ -271,7 +271,7 @@ var TopicEditorPage = function() {
 
   this.dragSkillToSubtopic = async function(skillDescription, subtopicIndex) {
     await waitFor.visibilityOf(
-      uncategorizedSkills.first(),
+      await uncategorizedSkills.first(),
```

Comment:
We don't have to "await" this because `first()` does not return a promise. See https://www.protractortest.org/#/api?view=ElementArrayFinder.prototype.first.

Reply:
Do we have to await for element.all() ?

Reply:
Nope. See https://www.protractortest.org/#/api?view=ElementArrayFinder.prototype.all

Reply:
so we don't have to await for element.all().first() too?

Reply:
Yup.